corehq/ex-submodules/pillowtop/tests/test_elasticsearch.py

```diff
@@ -91,7 +91,8 @@ def test_assume_alias(self):
         initialize_index_and_mapping(self.es, TEST_INDEX_INFO)
         doc_id = uuid.uuid4().hex
         doc = {'_id': doc_id, 'doc_type': 'CommCareCase', 'type': 'mother'}
-        send_to_elasticsearch(self.index, TEST_INDEX_INFO.type, doc_id, get_es_new, 'test', doc)
+        ElasticsearchInterface(get_es_new()).create_doc(
+            self.index, TEST_INDEX_INFO.type, doc_id, {'doc_type': 'CommCareCase', 'type': 'mother'}, False)
```

Comment:
why this change?

Reply:
I wanted to make `pillowtop.processors.elastic.send_to_elasticsearch` more private (preferring the one in `corehq.elastic.send_to_elasticsearch` instead. We could use the one `corehq.elastic.send_to_elasticsearch` here, but it requires mocking `ES_MET`, so I made this change.

Reply:
This should be fine, since `ElasticsearchInterface` is designed for public usage.